app/views/products/index.html.haml

```diff
@@ -1,13 +1,7 @@
 - if user_signed_in?  
   - if current_user.owner
-    %h1 Welcome owner!
+    %h1 Welcome owner!  
   - else
-    %h1 Today's Menu
+    %h2 Welcome user!
```

Comment:
`Welcome user!` Ddoes not seem like good UX

Reply:
This is a placeholder which is actually not related to the functionality we are implementing in this PR and hence is out of scope for this PR.